src/commands/git-metadata/__tests__/gitdb.test.ts

```diff
@@ -47,17 +48,66 @@ describe('gitdb', () => {
   })
   const testError = new Error('call failed')
 
+  // This is a bit hacky but the documentation of simpleGit asks you to depend on
+  // a behavior of VersionResult which is not debined in its typings: it has an
+  // override on toString. Here I am replicating this toString behavior so that
+  // the tests pass
```

Comment:
any chance we can add a permalink to where they're adding this `toString` behavior?

Reply:
```suggestion
  // the tests pass. See https://github.com/steveukx/git-js/blob/d184c13273abca4b6572c260f9625c19f944d4f7/simple-git/src/lib/tasks/version.ts#L15-L39
```